site/content/blogs/release-v129.en.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'AWS Copilot v1.29: Pipeline template overrides and CloudFront cache invalidation'
+twitter_title: 'AWS Copilot v1.29'
+image: ''
+image_alt: ''
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.29: Pipeline template overrides and CloudFront cache invalidation!
+
+Posted On: July 18, 2023
+
+The AWS Copilot core team is announcing the Copilot v1.29 release.
+
+Special thanks to [@tjhorner](https://github.com/tjhorner) and [@build-with-aws-copilot](https://github.com/build-with-aws-copilot), who contributed to this <release class=""></release>
+Our public [сommunity сhat](https://app.gitter.im/#/room/#aws_copilot-cli:gitter.im) is growing and has nearly 500 people online and over 2.9k stars on [GitHub](http://github.com/aws/copilot-cli/).
+Thanks to every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.29 brings big enhancements to help you develop more flexibly and efficiently:
+
+- **Pipeline overrides**: In [v1.27.0](https://aws.github.io/copilot-cli/blogs/release-v127/#extend-copilot-generated-aws-cloudformation-templates), we introduced CDK and yaml patch overrides for workload and environment CloudFormation templates. Now you can enjoy the same extensibility for Copilot pipeline templates! [See detailed section](#pipeline-overrides).
+- **Static Site enhancements**: We've improved our [latest workload type](https://aws.github.io/copilot-cli/blogs/release-v128/#static-site-service-type) with CloudFront cache invalidation and Static Site-tailored operational commands. [See detailed section](#static-site-enhancements).
+
+???+ note "What’s AWS Copilot?"
+
+The AWS Copilot CLI is a tool for developers to build, release, and operate production ready containerized applications on AWS.
+From getting started, pushing to staging, and releasing to production, Copilot can help manage the entire lifecycle of your application development.
+At the foundation of Copilot is AWS CloudFormation, which enables you to provision infrastructure as code.
+Copilot provides pre-defined CloudFormation templates and user-friendly workflows for different types of micro service architectures,
+enabling you to focus on developing your application, instead of writing deployment scripts.
```

Comment:
Just curious - without the extra indentation in front of these lines, does it render properly (i.e. they are under a "What's AWS Copilot banner" that's collapsible)

Reply:
good catch! TY.